src/lib/config.js

```diff
@@ -10,33 +10,47 @@ function isRunningTests () {
      process.argv[1].endsWith('mocha'))
 }
 
-function loadConfig () {
-  const localConfig = {
-    features: {
-      hasCreditAuth: Config.castBool(Config.getEnv(envPrefix, 'FEATURE_CREDIT_AUTH'))
-    }
+function useTestConfig () {
+  return !Config.castBool(process.env.UNIT_TEST_OVERRIDE) && isRunningTests()
+}
+
+function parseAmountConfig () {
+  return {
+    precision: parseInt(Config.getEnv(envPrefix, 'AMOUNT_PRECISION'), 10) || 10,
+    scale: parseInt(Config.getEnv(envPrefix, 'AMOUNT_SCALE'), 10) || 2
   }
+}
 
+function parseCurrencyConfig () {
+  return {
+    code: Config.getEnv(envPrefix, 'CURRENCY_CODE') || null,
+    symbol: Config.getEnv(envPrefix, 'CURRENCY_SYMBOL') || null
+  }
+}
+
+function parseAdminConfig () {
   const admin_user = Config.getEnv(envPrefix, 'ADMIN_USER') || 'admin'
   const admin_pass = Config.getEnv(envPrefix, 'ADMIN_PASS')
   const admin_fingerprint = Config.getEnv(envPrefix, 'ADMIN_TLS_FINGERPRINT')
 
   if (admin_pass || admin_fingerprint) {
-    localConfig.default_admin = _.omit({
+    return _.omit({
```

Comment:
It would be more clear to have an explicit `return undefined`; otherwise the reader is wondering if it is intentional or not.